drop `react-native-gesture-handler` in router (#24314)
expo/expo: docs/pages/router/advanced/drawer.mdx

```

> > After changing your `babel.config.js`, once run `npx expo start --clear` to start the development server and clear the babel cache.

Reviewer: ```suggestion
> After changing your **babel.config.js**, run `npx expo start --clear` once to start the development server and clear the babel cache.
```

Not related to PR (happy to create a follow up one too if required) but this is a small suggestion to apply our writing style guide for filenames and also update the verbiage about running the command as an instruction.

<Tabs>
<Tab label="SDK 49 and lower">

Now you can use the `Drawer` layout to create a drawer navigator.

```jsx app/_layout.js
import { Drawer } from 'expo-router/drawer';

export default function Layout() {
  return <Drawer />;
}
```

</Tab>

<Tab label="SDK 50 and greater">

> Now you can use the `Drawer` layout to create a drawer navigator. You'll also need to wrap the `<Drawer />` in a `<GestureHandlerRootView />` to enable gestures, this is required as of Expo Router v3 and greater. You only need one `<GestureHandlerRootView />` in your component tree, so any nested routes will not need to be wrapped.

Reviewer: ```suggestion
Now you can use the `Drawer` layout to create a drawer navigator. You'll need to wrap the `<Drawer />` in a `<GestureHandlerRootView>` to enable gestures. It is required as of Expo Router v3 and greater. You only need one `<GestureHandlerRootView>` in your component tree. Any nested routes are not required to be wrapped individually.
```

```jsx app/_layout.js
import { GestureHandlerRootView } from 'react-native-gesture-handler';
import { Drawer } from 'expo-router/drawer';

export default function Layout() {
  return (
    <GestureHandlerRootView style={{ flex: 1 }}>
      <Drawer />
    </GestureHandlerRootView>
  );
}
```

> Be careful about using `react-native-gesture-handler` on web as it can increase the JavaScript bundle size significantly. Learn more in [platform-specific modules](/router/advanced/platform-specific-modules/).

Reviewer: ```suggestion
> **Note:** Be careful when using `react-native-gesture-handler` on the web. It can increase the JavaScript bundle size significantly. Learn more in [platform-specific modules](/router/advanced/platform-specific-modules/).
```

Let's convert this in a callout as it seems like semi-warning or users should pay attention to.

</Tab>

> <Tabs>

Reviewer 2: ```suggestion
</Tabs>
```

Syntax fix.
